torch/testing/_internal/common_methods_invocations.py

```diff
@@ -865,6 +907,24 @@ def sample_inputs_fliplr_flipud(op_info, device, dtype, requires_grad):
            supports_tensor_out=False,
            test_inplace_grad=False,
            sample_inputs_func=sample_inputs_broadcast_to),
+    TriangularOpInfo('cholesky_inverse',
```

Comment:
We can probably remove the previous autograd test now:

https://github.com/pytorch/pytorch/blob/412e3f46e9fca591854d26ad007cb929ca87ec79/test/test_autograd.py#L2858

Reply:
Done.